src/routes/libraries.js

```diff
@@ -29,6 +29,8 @@ const browse = async (query, searchFields) => {
         batch: batch => {
             hits.push(...batch);
         },
+    }).catch(err => {
+        throw err instanceof Error ? err : new Error(`${err.name}: ${err.message}`);
```

Comment:
Sorry for the basic JS question: What can errors be instance of if not `Error`?

Also: what error/issue did we stumble upon that led us to add this? Is this related to the change in `libraries.spec.js` that does a try/catch for `latest`?